data/json/overmap/overmap_terrain/overmap_terrain_microlab.json

```diff
@@ -55,6 +55,50 @@
     "sym": "L",
     "copy-from": "microlab_sub_connector"
   },
+  {
+    "type": "overmap_terrain",
+    "id": "microlab_generic_hallway_start",
```

Comment:
Since the others are just doing a `copy-from` and not changing anything, you can just do `"id": [ "microlab_generic_hallway_start", "microlab_generic_firebreak", "microlab_generic_hallway", "microlab_generic_stairs_down",  ... and so on ]` to get the same result without so many lines.

Reply:
Oh thats nice!
Thanks for letting me know.